src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Controllers/DraftApprenticeshipController.cs

```diff
@@ -88,6 +90,35 @@ public async Task<IActionResult> Update(long cohortId, long apprenticeshipId, [F
             return Ok();
         }
 
+        [HttpGet]
+        [Route("{apprenticeshipId}/prior-learning-summary")]
+        public async Task<IActionResult> GetApprenticeshipPriorLearningSummary(long cohortId, long apprenticeshipId)
+        {
+            var command = new GetDraftApprenticeshipPriorLearningSummaryQuery(cohortId, apprenticeshipId);
```

Comment:
I think the `command` var here should be `query`.

Reply:
done